docs/docs/middlewares/path.md

```diff
@@ -0,0 +1,41 @@
+# Path
+This function returns the URL for the given route. 
+
+## Parameters
+```path(page, queries, locale)```
+
+parameter        | type           | required | description
+-----------------|----------------|----------|------------
+page             | string         | ✓        | The name of page (e.g. www.mysite.com).
```

Comment:
U-uh, www.mysite.com is canonical `url` provided by URL middleware. `page` is the key of current page listed in `site.yml`, for example `index` or `contact`.